app/views/Dashboard/AdvancedFilters/index.tsx

```diff
@@ -25,15 +30,68 @@ interface Thematic {
 }
 
 const ADVANCED_FILTER_OPTIONS = gql`
-query AdvancedFilterOptions($thematic: String!, $type: String!) {
+query AdvancedFilterOptions($thematics: [String!], $type: String) {
         filterOptions {
             types
             thematics(type: $type)
-            topics(thematic: $thematic)
+            topics(thematics: $thematics)
         }
     }
 `;
 
+interface ChipProps {
+    name: string;
+    value?: string[] | null | undefined;
+    onChange: React.Dispatch<React.SetStateAction<string[] | undefined>>;
+}
+
+function ChipLayout(props: ChipProps) {
```

Comment:
Is there a reason we didn't include this in the Chip component itself? It looks like a generic layout.

Reply:
Because I'm something of a developer myself :V